src/test/java/Homework17.java

```diff
@@ -0,0 +1,22 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework17 extends BaseTest {
+
+    @Test
+    public void addSongToPlaylist() throws InterruptedException {
+        String addedSongNotification = "Added 1 song into";
+        urlAccess();
+        getEmail("ponypony123@gmail.com");
+        getPassword("Testtesttest123123$$");
+        loginUser();
+        searchSong("Reactor");
+        searchResult();
+        selectFirstSong();
+        clickAddButton();
+        selectPlaylist();
+        Assert.assertTrue(getNotificationMessage().contains(addedSongNotification));
```

Comment:
Test failed! You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/221798960-74fe1020-354c-4fe4-b366-f1aa64704287.png)

![image](https://user-images.githubusercontent.com/104761482/221799057-557fce90-191e-4a1b-8138-524e6794b2e2.png)

**Hint**

Window (otificationMessage) not appears because the song (Reactor) has already been added to the playlist

![image](https://user-images.githubusercontent.com/104761482/221799170-708021b3-16d0-418d-b9f0-230a2b6b2332.png)